meeteval/viz/__main__.py

```diff
@@ -271,10 +286,22 @@ def add_argument(self, command_parser, name, p):
             if name == 'alignment':
                 command_parser.add_argument(
                     '--alignment',
-                    choices=['tcp', 'cp', 'tcp,cp', 'cp,tcp', 'tcorc', 'orc'],
-                    help='Specifies which alignment is used.\n'
-                         '- cp: Find the permutation that minimizes the cpWER and use the "classical" alignment.\n'
-                         '- tcp: Find the permutation that minimizes the tcpWER and use a time constraint alignment.'
+                    choices=['tcp', 'cp', 'orc', 'greedy_orc', 'tcorc', 'greedy_tcorc', 'greedy_dicp', 'greedy_ditcp'],
+                    nargs='+',
+                    help='Specifies which assigment and alignment are used. If a time-constrained algorithm is '
+                         'selected for the stream assignment, then a time-constrained alignment will be computed, '
+                         'otherwise the "classical" alignment without a time constraint is used.\n'
```

Comment:
I don't get it, why the time constraint alignment is used and where the difference to the assignment is.

Reply:
I simplified the help text